composer.json

```diff
@@ -1,15 +1,14 @@
 {
     "name": "privatebin/privatebin",
     "description": "PrivateBin is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted in the browser using 256 bit AES in Galois Counter mode (GCM).",
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/PrivateBin/PrivateBin"
-        }
-    ],
```

Comment:
You can still leave the repo information here, don't you?
They are only useful when we publish the project on https://packagist.org/ e.g., but that's another thing. (created https://github.com/PrivateBin/PrivateBin/issues/55 for it)

So basically this is just metadata.